app/scripts/lib/app-start.js

```diff
@@ -275,6 +275,7 @@ function (
         screenWidth: screenInfo.screenWidth,
         able: this._able,
         isSampledUser: isSampledUser,
+        uniqueUserId: this._getUniqueUserId(),
```

Comment:
Not for this PR, but we are now passing in both `able` and the `uniqueUserId`. We could move the [isSampledUser](https://github.com/mozilla/fxa-content-server/pull/2787/files#diff-b80f22cfd726dfdc2bb7ed3e82dc7264R257) logic into the Metrics module.